lib/rubycritic.rb

```diff
@@ -1,22 +1,7 @@
-require "rubycritic/configuration"
+require "rubycritic/command_factory"
 
 module Rubycritic
   def self.create(options = {})
```

Comment:
AFAIK RubyCritic never had a communicated, agreed on and versioned API. So this should be fine.
@guilhermesimoes what do you think about this?

Personally I have the guideline that unless a project has  a separate piece of documentation that says "API" then it doesn't have one I can rely on. This keeps my expectation in check.